testing/qt5-qtserialport/APKBUILD

```diff
@@ -0,0 +1,43 @@
+# Contributor: Denis Ryabyy <vv1r0x@gmail.com> 
+# Maintainer: Denis Ryabyy <vv1r0x@gmail.com>
+pkgname=qt5-qtserialport
+pkgver=5.6.0
+pkgrel=1
```

Comment:
It’s a new aport, so pkgrel should be 0. See [pkgrel on wiki](https://wiki.alpinelinux.org/wiki/APKBUILD_Reference#pkgrel).